.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -8,18 +8,23 @@ labels: ["feature request"]
 ## Feature Request
 
 ### Description
+
```

Comment:
due to prettier going over the entire codebase and formatting, many of the changes will be whitespace changes. the main files and directories to note of consequential change for this PR are:

.github/workflows
.husky/
README.md
hardhat.config.ts
package.json